modular_skyrat/master_files/code/modules/mob/mob_movement.dm

```diff
@@ -0,0 +1,18 @@
+/mob/living/carbon/verb/army_crawl()
+	set name = "Army Crawl"
+	set category = "IC"
+
+	var/mob/living/carbon/crawler = src
+	if(!crawler.resting)
+		return balloon_alert(src, "you must be laying down to army crawl.")
+	switch(HAS_TRAIT(crawler, TRAIT_PRONE))
+		if(FALSE)
+			balloon_alert_to_viewers("[crawler] begins to lower themself all the way to the ground.")
+			if(!do_after(crawler, 3 SECONDS))
```

Comment:
No checks for if we stand up in the meantime.

Reply:
you cannot stand up when in the prone state

Reply:
what if someone helps you up? Does it cancel?

Reply:
yes

Reply:
this is also outdated, but answer is still true